web/templates/confirm_dialog/confirm_subscription_invites_warning.hbs

```diff
@@ -1,5 +1,3 @@
 <p>
-    {{#tr}}
-    Are you sure you want to create stream ''''{stream_name}'''' and subscribe {count} users to it?
-    {{/tr}}
+    {{t "Are you sure you want to create stream ''''{stream_name}'''' and subscribe {count} users to it?" }}
```

Comment:
What is going on with the (existing) `''''` here? It seems surprising that this might be correct.

Reply:
`''` represents a single ASCII apostrophe ' ([documentation](https://formatjs.io/docs/core-concepts/icu-syntax/#quoting--escaping)). I’m not sure why I wrote this `''''` (in #18075), which displays as two ASCII apostrophes '', but fixing would require retranslation.

Reply:
Okay I have reconstructed my thought process. It was wrong.

(You do need to write `'{'''` to get {', though.)